packages/gateway/src/constants.ts

```diff
@@ -39,6 +39,9 @@ export const SENTRY_DSN = process.env.SENTRY_DSN
 export const PRODUCTION = process.env.NODE_ENV === 'production'
 export const QA_ENV = process.env.QA_ENV || false
 
+export const USER_NOTIFICATION_DELIVERY_METHOD =
+  process.env.USER_NOTIFICATION_DELIVERY_METHOD || 'sms'
+
```

Comment:
Is it being used anywhere in gateway?

Reply:
hmm. In validateNotificationDeliveryMethod function